src/content/guides/advanced-coredns-configuration/index.md

```diff
@@ -0,0 +1,73 @@
++++
+title = "Advanced CoreDNS Configuration"
+description = "Here we describe how you can customize the configuration of the managed CoreDNS service in your clusters"
+date = "2018-10-30"
+type = "page"
+weight = 50
+tags = ["tutorial"]
++++
+
+# Advanced CoreDNS Configuration
+
+The [CoreDNS addon](https://github.com/coredns/coredns) running inside your cluster has additional configuration options and features that can be customized.
+
+You can customize two of these configuration options on a per cluster basis through a ConfigMap inside your clusters. The ConfigMap is named `coredns-user-values` and is located in the `kube-system` namespace.
+
+__Note:__ This feature is only available in more recent cluster versions. To check if your cluster version supports customization through the ConfigMap, you can check if the above-mentioned ConfigMap is present.
+
+__Warning:__ Please do not edit any of the other CoreDNS related resources. Only the user ConfigMap is safe to edit.
+
+```nohighlight
+$ kubectl -n kube-system get cm coredns-user-values
+NAME                                   DATA      AGE
+coredns-user-values                    0         11m
+```
+
+On cluster creation the ConfigMap is empty and below-mentioned defaults will be applied to the final CoreDNS deployment. To customize any of the configuration options, you just need to add the respective line(s) in the data field of the user ConfigMap.
+
```

Comment:
Dore coreDNS read the configuration automatically and does a hot reload? maybe it is worth to mention

Reply:
chart-operator does the config update, and coredns then hot reloads AFAIK, but not sure we need to overload the technical details here

Reply:
well my concern is if the user needs to restart coredns

Reply:
user does not need to restart coredns